### ISSUE TITLE
Professional views client card

### ISSUE THREAD

**Issue by situatedbit** [loconomics/loconomics#308]
Currently when a professional views client details, they view a client editor form. Even if the client cannot be edited by the professional, the details are displayed in a form. The form looks like it can be edited. Additionally, when the client can be edited by the professional, the UI is mixed — the editable client details and the non-editable links out to special services. 

Additionally, there are smaller UI issues. For example, the delete client button is just as prominent as the save client details button, and it's very close to the save button.

A solution is to create a client card for viewing clients, and then a separate action for editing.

## Stories
- As a professional viewing and managing clients, I want to view client details together in one place. 
- I want to know if I can or can not edit client details.

This is a larger story requiring elaboration, and probably refinement into issues. I've made some sketches, highlighting issues with the current client editor form.

![client-card](https://cloud.githubusercontent.com/assets/6644016/23732431/ec73752a-0427-11e7-80b5-9525463d465a.png)

**Comment by joshdanielson** [comment]
@IagoSRL this is also an accessibility issue as it "appears" they can edit but can't (I don't know the exact part of the course it's mentioned in but know it is). So we will need a clientView activity. This can likely wait until after September 1st and something I may be able to do. I'm going to hide the "call" and "write email" buttons for now as they should be actual buttons but they'll only be used in the new clientView activity so better to wait until then.

**Comment by IagoSRL** [comment]
I agree. The idea to create a readable view is fine (although, the sketches show an editable notes field with 'save note'-'notes saved' button); the card view looks good, but we must go simpler now.

The source of the problem is the set of `.LightForm .form-control` styles, that replace default ones without take care of the 'disable' state. Usually, if the field is disable the background is darker with a `background-color: #cccccc;`.

@joshdanielson I will commit now a minor change to make disabled input fields under a LightForm class, to have that standard background, but please review that meets the contrast needed (we may need to change the default Bootstrap background too if no enough contrast).
